src/main/java/org/simulator/math/MaxDivergenceTolerance.java

```diff
@@ -0,0 +1,50 @@
+package org.simulator.math;
+
+import org.simulator.math.odes.MultiTable.Block.Column;
+
+public class MaxDivergenceTolerance extends QualityMeasure {
+
+    /**
+     * The metric the relative distance is based on
+     */
+    protected MaxAbsDistance metric;
+
+    /**
+     * Absolute tolerance for the reaction
+     */
+    protected double absTol;
+
+    /**
+     * Relative tolerance for the reaction
+     */
+    protected double relTol;
+
+    /**
+     * Default Constructor
+     */
+    public MaxDivergenceTolerance(double absTol, double relTol) {
+        super(Double.NaN);
+        metric = new MaxAbsDistance();
+        this.absTol = absTol;
+        this.relTol = relTol;
+    }
+
+    /* (non-Javadoc)
+     * @see org.sbml.simulator.math.Distance#distance(java.lang.Iterable, java.lang.Iterable, double)
```

Comment:
Document this better ! Please describe what the distance is and state the used formula in the comment. This is the core comparison function metric used. This must be documented so that others understand what is done here.

Reply:
Added the comments properly in the initial description of the class.